.travis.yml

```diff
@@ -3,20 +3,23 @@ dist: trusty
 
 language: scala
 
+addons:
+  firefox: latest
+  chrome: stable
+
 before_script:
   - "export DISPLAY=:99.0"
```

Comment:
I think it might be possible to simplify this with xvfb-run wrapper: https://docs.travis-ci.com/user/gui-and-headless-browsers/#using-the-xvfb-run-wrapper